torch/csrc/dynamo/compiled_autograd.h

```diff
@@ -358,9 +356,13 @@ class CompiledNodeArgs {
     for (auto& i : fn->post_hooks()) {
       i->compiled_args(*this);
     }
+    if (fn->tensor_post_acc_grad_hooks() != nullptr) {
+      fn->tensor_post_acc_grad_hooks()->compiled_args(*this);
+    }
```

Comment:
Since this only applied to accumulate grad nodes, it should go in accumulate_grad.cpp